website/docs/layouts/flex/partials/code/how-to-use.md

```diff
@@ -0,0 +1,573 @@
+
+!!!
+
+### Media + Text/Content
+
+This is a classic layout, where a "media" element (it could be an image, an icon, an avatar, etc) is laid out next to a block of text (but could be more complex content). The two elements are visually centered along their horizontal axis.
+
+
+```handlebars
+<Hds::Layout::Flex @align="center" @gap="8">
+  <img class="doc-flex-media-avatar" src="/assets/images/avatar.png" alt="portrait of a cat wearing coat and tie" />
+  <Hds::Text::Body @size="200" @tag="p">Lorem ipsum dolor sit amet</Hds::Text::Body>
+</Hds::Layout::Flex>
+
+<br/>
+
+<Hds::Layout::Flex @align="center" @gap="8">
+  <Hds::Icon @name="info" @size="24" />
+  <Hds::Text::Body @size="200" @tag="p">Lorem ipsum dolor sit amet</Hds::Text::Body>
+</Hds::Layout::Flex>
+
+<br/>
+
+<Hds::Layout::Flex @align="start" @gap="16">
+  <Hds::IconTile @logo="boundary" />
+  <Hds::Layout::Flex @direction="column" @gap="4">
+    <Hds::Text::Display @size="400" @tag="h5">Lorem ipsum dolor</Hds::Text::Display>
+    <Hds::Text::Body @size="200" @tag="p">Lorem ipsum dolor sit amet consectetur adipiscing elit</Hds::Text::Body>
+    <Hds::Link::Standalone @icon="external-link" @iconPosition="trailing" @text="Documentation" @href="#" />
+  </Hds::Layout::Flex>
+</Hds::Layout::Flex>
+```
+
+### Group of cards
+
+Using the automatic layout offered by flexbox, it's extremely easy to create a group of identically sized, evenly spaced cards:
+
+```handlebars
+<Hds::Layout::Flex @align="stretch" @gap="48" as |LF|>
+  <LF.Item @basis={{0}} @grow={{true}}>
+    <Hds::Card::Container @level="mid" @hasBorder={{true}} class="doc-flex-cards-item">
+      <Hds::Text::Body @tag="p" @size="200">This is some generic content inside a card</Hds::Text::Body>
+    </Hds::Card::Container>
+  </LF.Item>
+  <LF.Item @basis={{0}} @grow={{true}}>
+    <Hds::Card::Container @level="mid" @hasBorder={{true}} class="doc-flex-cards-item">
+      <Hds::Text::Body @tag="p" @size="200">This is some other generic content inside a card</Hds::Text::Body>
+    </Hds::Card::Container>
+  </LF.Item>
+  <LF.Item @basis={{0}} @grow={{true}}>
+    <Hds::Card::Container @level="mid" @hasBorder={{true}} class="doc-flex-cards-item">
+      <Hds::Text::Body @tag="p" @size="200">This is also some generic content inside a card, but the text is longer and makes the card grow vertically</Hds::Text::Body>
+    </Hds::Card::Container>
+  </LF.Item>
+</Hds::Layout::Flex>
+```
+
+!!! Warning
+
+A responsive implementation of this UI pattern is a bit more complex, and requires to use `min-max/max-width` values for the cards, and allow wrapping of the flex item elements. It may also require changing direction of the flex container at a certain breakpoint.
```

Comment:
```suggestion
A responsive implementation of this UI pattern is a bit more complex and requires usage of `min-max/max-width` values for the cards and allowing wrapping of the flex item elements. It may also require changing the direction of the flex container at a certain breakpoint.
```